pg-index-health/src/test/java/io/github/mfvanek/pg/support/statements/AbstractDbStatement.java

```diff
@@ -13,18 +13,30 @@
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.sql.Statement;
-import java.util.Objects;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicReference;
 import javax.annotation.Nonnull;
 
-abstract class AbstractDbStatement implements DbStatement {
+import static io.github.mfvanek.pg.model.PgContext.DEFAULT_SCHEMA_NAME;
 
-    protected final String schemaName;
+public abstract class AbstractDbStatement implements DbStatement {
```

Comment:
Try to remove `public` here

Reply:
If setSchemaName method and AbstractDbStatement are set as protected or default as controllers, they cannot be referenced from DataBasePopulator of other packages.

Shall we introduce the facade pattern?

Reply:
> If setSchemaName method and AbstractDbStatement are set as protected or default as controllers, they cannot be referenced from DataBasePopulator of other packages.
> 
> Shall we introduce the facade pattern?

Oh, I missed this moment. Thank you. Let's leave it as it is